src/sdk/namespace_fs.js

```diff
@@ -59,6 +59,41 @@ function isDirectory(ent) {
     }
 }
 
+/**
+ * 
+ * @param {*} stat - entity stat yo check
+ * @param {*} fs_account_config - account config using to check symbolic links
+ * @param {*} entry_path - path of symbolic link
+ * @returns 
+ */
+async function isDirectoryOrSymLinkToDirectory(stat, fs_account_config, entry_path) {
```

Comment:
in general change the signatures of new functions to snake case to be more consistent with the rest of the code
```suggestion
async function is_directory_or_symlink_to_directory(stat, fs_account_config, entry_path) {
```